src/components/ErrorList/ErrorList.tsx

```diff
@@ -0,0 +1,72 @@
+import { useRef, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+import validatorErrors from "../../../public/locales/en/validatorErrors.json";
+import useWindowSize from "../../helpers/useWindowSize";
+import IconError from "../Icon/icons/IconError";
+import { ScrollContainer, Container } from "../TokenList/TokenList.styles";
```

Comment:
You're using styled components from TokenList. We should refactor this and move ScrollContainer and Container to Overlay/Overlay.styles.